src/DynamoUtilities/DynamoPathManager.cs

```diff
@@ -364,10 +384,13 @@ public static bool PreloadAsmVersion(string version, DynamoPathManager pathManag
             Debug.WriteLine(string.Format("Attempting to preload ASM version {0}", version));
 
             string hostLocation;
-            if (!FindAsm(version, out hostLocation))
+            if (!FindAlternativeAsm(version, pathManager, out hostLocation))
```

Comment:
Yes, that's right, and it's part of the bigger change that will eventually delegate the responsibility of path determination to host application. More on that in later PRs.